apps/savingthrow.js

```diff
@@ -570,10 +578,12 @@ export class SavingThrow {
             let tokens = Object.keys(flags)
                 .filter(key => key.startsWith('token'))
                 .map(key => flags[key]);
+            /*let tokenOrActor = await fromUuid(message.uuid);
+            let actor = tokenOrActor.actor ? tokenOrActor.actor : tokenOrActor;*/
 
             let ids = tokens.filter(t => {
                 if (t.roll != undefined) return false;
-                let actor = game.actors.get(t.actorid);
+                let actor = game.actors.get(t.actorid);  //Haven't changed this one as I am not clear about this check.

```

Comment:
This is a check to make sure the Actor exists before trying to roll.  Since it needs the Actor to run some functions, there's no reason to roll if it can't actually do the roll.  So this would have to be an actual Actor.